src/search/constants.js

```diff
@@ -0,0 +1,3 @@
+export const COPROCURE_STATIC_PATH = "https://www.coprocure.us/static/";
+export const LOGO_PATH = COPROCURE_STATIC_PATH+"img/logo-svg.svg";
```

Comment:
The Javascript `URL` object might be preferable, in case there are clashing slashes.